src/locales/en/translation.json

```diff
@@ -9,28 +9,28 @@
     "dropdowns": [
       {
         "buttonLabel": "Nav Label 1",
```

Comment:
We didn't get translations for nav and parent link labels?

Reply:
The translator kept it the same, leaving a comment regarding the parent link labels: "This is a coding term I think developers call this thing the same in Spanish."